hummingbot/core/utils/trading_pair_fetcher.py

```diff
@@ -54,14 +63,19 @@ def __init__(self):
                         # https://api.binance.com/api/v1/exchangeInfo
                         if "123456" in raw_trading_pairs:
                             raw_trading_pairs.remove("123456")
-                        return [BinanceMarket.convert_from_exchange_trading_pair(p) for p in raw_trading_pairs]
+                        trading_pair_list: List[str] = []
+                        for p in raw_trading_pairs:
+                            if BinanceMarket.convert_from_exchange_trading_pair(p) is not None:
+                                trading_pair_list.append(p)
+                            else:
+                                self.logger().warning(f"Trading Pairs are not updated for binance for pair: {p}")
```

Comment:
This log message is difficult to understand for both users and even our engineers.

Change it to

f"Could not parse the trading pair {p}, skipping it..."